six/two/two.cpp

```diff
@@ -511,6 +498,11 @@ char **Two::getCheckWarnings(SixPyObject *check)
     }
 
     Py_ssize_t numWarnings = PyList_Size(warns_list);
+    if (numWarnings == -1) {
```

Comment:
This isn't very well documented - I checked the source and this is correct but could we please add a small comment?

Reply:
done, with link to source